CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `TestClusterTooOld` for testing installations
```

Comment:
There's more in this PR than just `TestClusterTooOld`

Reply:
You are right and this change is already in master. I will update the CHangelog